resources/startPage.html.template

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src {{ cspSource }} file:; script-src 'nonce-{{ nonce }}'; font-src {{ codiconsFontUri }}; style-src {{ cspSource }} {{ codiconsStyleUri }} 'nonce-{{ nonce }}';" />
+    <link href="{{ codiconsStyleUri }}" rel="stylesheet" />
+    <title>Docker - Get Started</title>
+
+    <style nonce="{{ nonce }}">
+        #titleBar {
+            align-items: center;
+        }
+
+        #titleBar h1 {
+            margin-block-start: .2em;
+            margin-block-end: .2em;
+        }
+
+        .icon .codicon {
+            font-size: 70px;
+            padding: 5px;
+        }
+
+        .icon:hover {
+            color: var(--vscode-editor-foreground);
+        }
+
+        #dockerIcon {
+            width: 70px;
+            padding: 5px;
+        }
+
+        #dockerIcon path {
+            fill: var(--vscode-editor-foreground);
+        }
+
+        .row {
+            display: flex;
+            flex-direction: row;
+            justify-content: flex-start;
+            align-items: flex-start;
+        }
+
+        .row>* {
+            margin: 10px;
+        }
+
+        .linkheading {
+            text-decoration: none;
+            color: var(--vscode-editor-foreground);
+        }
+    </style>
+</head>
+
+<body>
+    <div class="container">
+        <div class="row" id="titleBar">
+            <div>
+                <a href="https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker" tabindex="-1"><img src="{{ dockerIconUri }}" width="80" alt="Docker logo" /></a>
+            </div>
+            <div>
+                <a href="https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker" class="linkheading">
+                    <h1>Docker Extension</h1>
+                    <span>by Microsoft</span>
+                </a>
+            </div>
+        </div>
+        <div class="row">
+            <div>
+                <a href="command:vscode-docker.configure" class="linkheading icon" tabindex="-1">

```

Comment:
Will telemetry capture the fact that this command was invoked from the start page?

Reply:
No, unfortunately. It's not possible to pass any arguments to the command so it will be completely indistinguishable from one launched by the palette.

Reply:
That's a bummer. Maybe it is worth opening an issue about it against VS Code core?

Reply:
Good news, I'm wrong! You can pass arguments in the URI: https://code.visualstudio.com/api/extension-guides/command#using-commands

Reply:
Ooh, that is fantastic! So we can track what users click on the start page, that's awesome.